tests/constraints.txt

```diff
@@ -14,7 +14,7 @@ fastjsonschema~=2.17.1
 tensorflow>=2.5,<2.15.0
 test-generator==0.1.2
 requests>=2.25.1
-opencv-python>=4.5
+opencv-python-headless>=4.5
```

Comment:
why do we need to use headless version?

Reply:
it was a test only) installation of opencv-python-headless should take less time. But I saw errors, reverted